osp-backend-docusaurus/docs/Maintainer-Guidelines.md

```diff
@@ -0,0 +1,19 @@
+---
+id: Maintainer-Guidelines
+title: Maintainer Guidelines
+---
+
+## Guidelines
+Make sure contributors are respecting the Contribution Guidelines. If they don't let them know about it: what is missing, what was disrespected?
+
+## Code Review
+- When possible get a second opinion before merging.
+- Make sure the PR has a link to the issue.
+- If you can, let them know why your change request makes sense.
+- (nice to have) If you have time, thank them for contributing to the project.
+- **Make sure that, if the PR changes actual code, to properly test it or wait for someone to confirm the behavior**
+
+## Merging PRs
+- Minimum 2 approvals are needed to get a PR merged
+- Select **Squash and Merge**.
+- Edit commit if necessary to follow our [Commit Message style guide](https://github.com/anitab-org/open-source-programs-web/wiki/Commit-Message-Style-Guide) and leave the PR id in the message.
```

Comment:
Please update the link to `Commit Message Style Guide` here as well as mentioned above.